tests/src/OneLogin/saml2_tests/idp_metadata_parser_test.py

```diff
@@ -51,7 +51,7 @@ def testParseRemote(self):
 
         data = OneLogin_Saml2_IdPMetadataParser.parse_remote('https://www.testshib.org/metadata/testshib-providers.xml')
         self.assertTrue(data is not None and data is not {})
-        expected_data = {'sp': {'NameIDFormat': 'urn:mace:shibboleth:1.0:nameIdentifier'}, 'idp': {'singleSignOnService': {'url': 'https://idp.testshib.org/idp/profile/SAML2/Redirect/SSO'}, 'entityId': 'https://idp.testshib.org/idp/shibboleth'}}
+        expected_data = {'sp': {'NameIDFormat': 'urn:mace:shibboleth:1.0:nameIdentifier'}, 'idp': {'singleSignOnService': {'url': 'https://idp.testshib.org/idp/profile/SAML2/Redirect/SSO', 'binding': 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect'}, 'entityId': 'https://idp.testshib.org/idp/shibboleth'}}
         self.assertEqual(expected_data, data)
 
     def testParse(self):
```

Comment:
I've pulled in one more metadata XML (from oktadev) and covered all combinations of the `required_slo/sso_binding` parameters to the `parse()` method.